README.md

```diff
@@ -14,51 +14,15 @@ The AKS Landing Zone Accelerator is only concerned with what gets deployed in th
-
-### AKS Construction Helper
-
-A flexible templating approach using Bicep that enables multiple scenarios using a Web based tool. It provides tangible artifacts to **quickly** enable AKS deployments through CLI or in your CI/CD pipeline.
-
-Driving the configuration experience is a [wizard](https://azure.github.io/AKS-Construction/?default=es) to guide your decision making, it provides presets for the main Azure Landing Zone deployment modes (Sandbox, Corp & Online). The output of this wizard experience are the parameters and CLI commands to immediately deploy using our maintained Bicep template to deploy your customized AKS environment in one step.
-
-Use [this repo](https://github.com/Azure/Aks-Construction) if you would like to use a guided experience to rapidly create your environment with a maintained Bicep template based on the architecture of the AKS Secure Baseline.
-
-<!-- ### Baseline Automation Module
-
-This reference implementation demonstrates recommended ways to automate the deployment of the components composing a typical AKS solution. This repository includes information about separation of duties (different teams managing different parts of the deployment process), CI/CD and GitOps best practices. 
-
-Use [this repo](https://github.com/Azure/aks-baseline-automation) if you would like to learn how to quickly setup and get access to templates to help setup your own DevOps environments for AKS workloads.  -->
-
-## Next Steps to implement AKS Landing Zone Accelerator
-Pick one of the options below
-
-### Follow a scenario driven tutorial within this repo
-
-Pick one of the scenarios below to get started on a reference implementation. For the AKS secure baseline with non-private cluster, use the [AKS baseline](https://github.com/mspnp/aks-baseline) reference implementation.
+1. This one. To Deploy our AKS Secure Baseline Scenario, Check out [AKS Secure Baseline](./Scenarios/AKS-Secure-Baseline-PrivateCluster/). Use [this repo](https://github.com/Azure/AKS-Landing-Zone-Accelerator/tree/main/Scenarios/AKS-Secure-Baseline-PrivateCluster) if you would like step by step guidance on how to deploy secure and well architected AKS clusters using our scenario based model and/or you are new to AKS or IaC. This model promotes a separation of duties, modularized IaC so you can pick and choose components you want to build with your cluster and has implementations in ARM, Terraform and Bicep. It is the best starting point for people new to Azure or AKS.
```

Comment:
Does "**This one.**" mean anything in the 1st item? Unfortunately, I didn't get it, so I asked for clarification that if it adds nothing meaningful to the sentence, we can delete that phrase.